### DISCUSSION TITLE
How to use clustering visualization in Grasshopper?

### DISCUSSION THREAD

**Discussion by eastnoob** [Tunny-gh/Tunny_in_Open_Sea#188] [General]
I have finished the optimization process and all other visualizations methods work well except clustering. I have tried selecting objectives and clicking "Show" buttons, and even tried all the buttons but they didn't work, such as popping up a chart. I do not know why so can you tell me how to use It  (˶ᵔ ᵕ ᵔ˶) ?

**Comment by hrntsm** [comment]
Thanks @eastnoob for your comment.
I checked and it seems to be a bug. Please wait a moment while I fix it.

**Comment by hrntsm** [comment]
I fix it in #255 
This update will include v0.11
